components/WriteAffirmationCardModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import Modal from "@mui/material/Modal";
+import '@/app/css/Affirmation.css';
+
+interface WriteAffirmationCardModalProps {
+  open: boolean;
+  handleModalClose: () => void;
```

Comment:
This can also be "on" instead of "handle" I think

Reply:
Done.